ui/redux/reducers/reactions.js

```diff
@@ -5,17 +5,20 @@ import * as REACTION_TYPES from 'constants/reactions';
 
 const defaultState = {
   fetchingReactions: false,
-  reactionsError: undefined,
   reactionsById: {},
 };
 
 export default handleActions(
   {
-    [ACTIONS.REACTIONS_LIST_STARTED]: state => ({ ...state, fetchingReactions: true }),
-    [ACTIONS.REACTIONS_LIST_FAILED]: (state, action) => ({
-      ...state,
-      reactionsError: action.data,
-    }),
+    [ACTIONS.REACTIONS_LIST_STARTED]: (state) => ({ ...state, fetchingReactions: true }),
+    [ACTIONS.REACTIONS_LIST_FAILED]: (state, action) => {
+      const { claimId } = action.data;
+
+      const newReactionsById = Object.assign({}, state.reactionsById);
+      newReactionsById[claimId] = 0;
+
+      return { ...state, fetchingreactions: false, reactionsById: newReactionsById };
```

Comment:
:small_red_triangle: (?) In `master`, we show the placeholder when the reaction-fetch fails.  With this change, it defaults to zero, which feels like a regression where the "don't incorrectly show 0 count" issue is re-opened.

My vote is the `master` behavior is the lesser evil.